File: Part1/calwebserver/src/calwebserver/Cal.java
package calwebserver;
import javax.jws.*;

@WebService
public class Cal {
	public double cal(double first_num,double sec_num, int operator){
		double result = 0.0;

		 switch (operator)  {
	        case 1:
	        	result=first_num + sec_num;
	            break;

	        case 2:
	        	result=first_num - sec_num;
	            break;

	        case 4:
	        	result=first_num / sec_num;
	            break;

	        case 3:
	        	result=first_num * sec_num;
	            break;
		 }
		 System.out.println(result);
		 return result;
	}
}

File: Part1/cal/public/angularjs/cal.js
var cal = angular.module('cal', []);

cal.controller('cal', function($scope, $http) {

	$scope.data = {
	    availableOptions: [
	      {value: 1, name: 'Add'},
	      {value: 2, name: 'Subtraction'},
	      {value: 3, name: 'Multiple'},
	      {value: 4, name: 'Division'}
	    ],
	    selectedOption: {value: 1, name: 'Add'} //This sets the default value of the select in the ui
    };

	$scope.submit = function() {
		$http({
			method : "POST",
			url : '/cal',
			data : {
				"first_num" : $scope.first_num,
				"sec_num" : $scope.sec_num,
				"action" : $scope.data.selectedOption.value
			}
		}).success(function(data) {

			if (data.statusCode == 200) {
				$scope.result = data.result;

			}else{
				$scope.result = data.result;
				alert("plsase check your itput");
			}

		}).error(function(error) {
			alert(error);
		});
	};
})
